Media/Translations/Default.xml

```diff
@@ -2899,4 +2912,28 @@
 		<nl>brandstof</nl>
 		<pl>Paliwo</pl>
 	</segment>
-</resources>
\ No newline at end of file
+	<segment>
```

Comment:
Please use the words "Checkpoint" and "Checkpoints" for better understanding for inexperienced players.

Reply:
Done